packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,204 @@
+| description   | `string`, `JSX`                                             | `undefined` | Body content                                                                               | ✅       |
+| title         | `string`                                                    | `undefined` | A brief heading                                                                            |          |
+| isDismissable | `boolean`                                                   | `true`      | Shows a dismiss button                                                                     |          |
+| onDismiss     | `function`                                                  | `undefined` | Callback when the dismiss button is pressed                                                |          |
+| contrast      | `high`, `low`                                               | `low`       | Can be set to `high` for more prominent look _(not related to a11y)_                       |          |
+| isFullWidth   | `boolean`                                                   | `false`     | Spans the entire width of container, otherwise max width is restricted to 584px by default |          |
+| variant       | `info`, `positive`, `notice`, `negative`                    | `info`      | Sets the color tone of entire alert. Icon is set automatically based on variant            |          |
+| isBorderless  | `boolean`                                                   | `false`     | Removes borders and border radii, useful for creating a full bleed layout                  |          |
+| actions       | `{ primary: PrimaryAction, secondary: SecondaryAction }` \* | `{}`        | Renders a primary action button and a secondary action link button                         |          |
+
+`PrimaryAction` and `SecondaryAction` will accept objects with the following keys:
+
+### PrimaryAction
+
+Renders and accepts a subset of `Button` props as keys:
+
+| Key       | Type       | Default     | Description | Required |
+| --------- | ---------- | ----------- | ----------- | -------- |
+| `text`    | `string`   | `undefined` | Content     | ✅       |
+| `onClick` | `function` | `undefined` | Callback    |          |
+
+### SecondaryAction
+
+Renders and accepts a subset of `Link` (button) props as keys:
+
+| Key       | Type       | Default     | Description | Required |
+| --------- | ---------- | ----------- | ----------- | -------- |
+| `text`    | `string`   | `undefined` | Content     | ✅       |
+| `href`    | `string`   | `undefined` | Hyperlink   |          |
+| `onClick` | `function` | `undefined` | Callback    |          |
+
+There'll also be an internal `BaseAlert` component that handles rendering web or native version similar to other Blade components. API would be similar to `Alert`.
+
+## a11y
```

Comment:
Will we use LiveAnnouncer for this? 

// @anuraghazra

Reply:
Not really needed. Tho in android we might need to since last time I checked the aria role alert did not announce the changes. Otherwise role=alert should suffice.

Reply:
From the React Native [docs](https://reactnative.dev/docs/accessibility#accessibilityrole) it should work ideally but yes if it doesn't we can definitely do this as an enhancement.

Reply:
if the alert is something which is shown dynamically based on some validation fetched from API call, then shouldn't we announce them with live-announcer? that was the use case for live announcer right?

Reply:
`role="alert"` would do that automatically, wouldn't it? Also we'll make it `polite` for blue and green alerts.